src/Command/CommandBase.php

```diff
@@ -805,18 +805,19 @@ protected function determineCloudEnvironment() {
   /**
    * Determine the Cloud application.
    *
-   * @param bool $link_app
+   * @param bool $prompt_link_app
+   * @param bool $prompt_app
    *
    * @return string|null
    * @throws \Exception
    */
-  protected function determineCloudApplication($link_app = FALSE): ?string {
-    $application_uuid = $this->doDetermineCloudApplication();
+  protected function determineCloudApplication($prompt_link_app = FALSE, $prompt_app = TRUE): ?string {
```

Comment:
`$prompt_app` argument appears to never be passed and can be removed. Same for `doDetermineCloudApplication`.